src/extension/_locales/en/messages.json

```diff
@@ -363,21 +394,36 @@
     "error_preview": {
       "message": "Preview generation failed. Please try again later."
     },
+    "error_preview_400": {
+      "message": "Preview generation failed. Check the file format, name or path."
+    },
     "error_preview_404": {
-      "message": "404 Not found: Source document does not exist, preview generation failed."
+      "message": "Source document does not exist, preview generation failed."
+    },
+    "error_preview_415": {
+      "message": "File type not supported. Refer to the documentation for supported file types."
```

Comment:
Maybe not for this PR.. but maybe we could have a `url` property defined here and a way to markup the message that should be used as the hyperlink for the message.

I.E 

```
"error_preview_415": {
      "message": "File type not supported. Refer to the [documentation] for supported file types."
      link: "https://www.aem.live/docs/limits"
    },
 ```
 
 Might be a problem for the translators tho.. nice way to address the issue though..

Reply:
Like the idea, worth trying out in a separate PR.